lib/cli/commands/init.js

```diff
@@ -1,5 +1,11 @@
 // Init
 import baseMiddleware from "../middlewares/base.js";
+import {getLogger} from "@ui5/logger";
+import {cleanLogOutput} from "../utils/logHelper.js";
```

Comment:
Import can't be resolved

Reply:
Removed. Please check the comments bellow: 
- https://github.com/SAP/ui5-cli/pull/686#issuecomment-1966144261
- https://github.com/SAP/ui5-cli/pull/686#issuecomment-1966154218